styles/ollama/catppuccin.user.css

```diff
@@ -0,0 +1,198 @@
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+    @icon-filter: @catppuccin[@@lookup][@icon_filter];
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+    & when not (@lookup = latte) {
+      img[src="/public/ollama.png"], img[src="https://ollama.com/public/assets/c889cc0d-cb83-4c46-a98e-0d0e273151b9/42f6b28d-9117-48cd-ac0d-44baaf5c178e.png"] {
+filter: @icon-filter;
+// background-color: @red;
+      }
+    }
+    .text-black, .text-black  * {
+      color: @text ;
+    }
+    .text-white {
+      color: @text;
+    }
+    .text-gray-900 {
+      color: @subtext1;
+    }
+
+    .bg-black {
+      background-color: @surface0;
+    }
+    .bg-neutral-50 {
+      background-color: @surface0;
+    }
+    .bg-green-50 {
+      background-color: @green;
+    }
+    .text-green-700 {
+      color: darken(@green, 30%)
+    }
+    li::marker {
+      color: @text !important;
+    }
+    .divide-gray-200 > * + * {
+border-color: @accent-color !important;
+    }
+    .border {
+      border-color: @accent-color;
+    }
+    .divide-neutral-200 > * + * {
+      border-color: @accent-color !important;
+          }
+    .border-neutral-200 {
+      border-color: @accent-color;
+    }
+    ul {
+      color: @text !important;
+    }
+    #readme > *, #display > *, #file-explorer > *, #readme, #display  {
```

Comment:
Are there better selectors we can use to avoid using `*`?

Reply:
> Are there better selectors we can use to avoid using `*`?

unfortunately i use the `*` selector as the elements are not under any class names which makes it hard to set colors for it.